compiler/src/dmd/cli.d

```diff
@@ -466,6 +466,26 @@ dmd -cov -unittest myprog.d
 
              $(P Note that multiple `-i=...` options are allowed, each one adds a pattern.)}"
         ),
+        Option("identifiers=<table>",
+            "Specify the non-ASCII tables for D identifiers",
+            `Set the identifier table to use for the non-ASCII values.
+                $(UL
+                    $(LI $(I UAX31): UAX31)
+                    $(LI $(I c99): C99)
+                    $(LI $(I c11): C11)
+                    $(LI $(I all): All, the least restrictive set, which comes all others (default))
```

Comment:
What does 'comes all others' mean? I'm no native speaker, but it doesn't make sense to me.

Reply:
Looks like I missed out a word "with".